optimum/exporters/openvino/__main__.py

```diff
@@ -100,6 +104,7 @@ def main_export(
     stateful: bool = True,
     convert_tokenizer: bool = False,
     library_name: Optional[str] = None,
+    model_loading_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
why add a new argument instead of using `model_kwargs` which I believe is passed to the model's `from_pretrained`